utils/versionparser.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	VersionSeparator = "."
```

Comment:
I'm pretty sure there was a reason for them being exported when I implemented this three months ago — No idea anymore ;)
Updated, constants are no longer exported.